### PR TITLE
Optz translate & add NSFW_zh-cn

### PR DESCRIPTION
**Pull request by KevinZonda**

### CODE REVIEW

**Comment by KevinZonda** [comment]
@me-shaon Plz CR.

File: translations/NSFW_LICENSE_zh-CN [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+任何其它操作，但风险自负。
+
+作者对这个项目中的代码一无所知。
+代码处于可用或不可用状态，没有第三种情况。
+
+
+        GOOD LUCK WITH THAT SHIT 公共许可证
+            复制、分发和修改的条款和条件
+
+0：在不导致作者被指责或承担责任的情况下，你可以做任何你想
```
Comment (b1f6c1c4) [inline translations/NSFW_LICENSE_zh-CN:14]: Not NSFW enough. In the [original text](https://github.com/me-shaon/GLWTPL/blob/master/NSFW_LICENSE#L17-L19):
```
  0. You just DO WHATEVER THE FUCK YOU WANT TO as long as you NEVER LEAVE
A FUCKING TRACE TO TRACK THE AUTHOR of the original product to blame for
or held responsible.
```

Comment (KevinZonda) [reply translations/NSFW_LICENSE_zh-CN:14]: I will make it more NSFW after going home😂😂😂

**Comment by me-shaon** [comment]
@KevinZonda any update on this PR? :)

**Pull request closed without merging**